public/sass/elements/forms/_form-validation.scss

```diff
@@ -10,7 +10,8 @@
 
   // Form inputs should have a red border
   // Add a red border to .form-controls that are direct descendants
-  > .form-control {
+  > .form-control,
+  > fieldset > .form-date > .form-group > .form-control {
```

Comment:
It does need to be quite specific so not to trigger the border on fields that are in a panel where the parent question has an error on it eg...

- yes/no question that reveals a panel with a date field in when yes is clicked.
- a validation error is triggered on the yes/no question when its not selected.
- when the panel is revealed the date fields inherit the border if its `.error .form-date .form-control`

could be trimmed down to `> fieldset > .form-date .form-control`?

ive updated the pull request